.travis.yml

```diff
@@ -103,6 +104,7 @@ before_script:
     - etc/travis/run-suite before_script "${SYLIUS_SUITE}"
 
 script:
+    - if [[ $CODING_STANDARD != "" ]]; then bin/ecs check src tests; fi
```

Comment:
Could we remove this check? We had checked CS in CI before, but it used to discourage contributors with failing builds just because of CS.

Reply:
We want to minimise contributors' efforts as much as possible and keeping coding standard isn't critical enough to prevent merging PRs. Project maintainers can run the command that fixes CS and commit necessary changes anytime, so that contributors do not have to even know about our tooling (therefore making contributions easier).

Reply:
It was around 2-3 years ago, so it's quite difficult to find these PRs. These commits look like this one here - https://github.com/Sylius/Sylius/commit/8c351120dcc63fb0a813f56faaae5f4174900463, I'm usually doing these before each release, it's not automated yet but I'd like to include that in release tool.

Reply:
Thanks for explaining, I'll consider this approach myself